src/rules/no-commonjs.js

```diff
@@ -35,6 +56,7 @@ const schemaObject = {
   properties: {
     allowPrimitiveModules: { 'type': 'boolean' },
     allowRequire: { 'type': 'boolean' },
+    allowConditionalRequire: { 'type': 'boolean' },
```

Comment:
```suggestion
    allowConditionalRequire: { 'type': 'boolean', default: true },
```
(altho for back compat we can't rely on the default getting applied)